python/doc/examples/functional_modeling/logistic_growth_model.ipynb

```diff
@@ -0,0 +1,305 @@
MjKJ5PI3KfxNMkdJ5W9SOEs2AHZo6UUgI4ngecbQj/Z0xU/07AlaDTIyZooxkImRTAyWxVhtfeG2WVuK3xioMERhQ9embn6rQ3yzCATVSMyBz7cqkb0VRiWVywtjIaygAD47BQvPQI4Oovzgp1EwrUOtEb2XQ5Yh/mA+v/2SxLJVwVCPIUo/vxp2lpXBtm3Ia9ai/yOWosQIiqTxFKt+xGxyQONuwGWCBp9x4DIOtO2aV8UGI0bOcYKTFtE7yX7iOYYRAxq0RNCLvgznbl6gKwPoQJdm37tXip7jpPE3yRbhS+EoqRSjA8ATR3oRyGR60osg+hJMF9qhaebPSyCojoyMiYJ6CZ2BTGTLZ+AiGrT4oMEXLb7YEYETkWjxRTJ6c2hvIt0H+jfJc2vtCAEUCFBkbPMp+HQL/HEUnO3gyREwY7CRdpsT4dZYpVSbpy3cFwEPd1Eie6+QshLYt7acFStyOLDFE326OyYHd8qHG7HLMqErVIF8aZdclYTNZjMcPw6bNmFYtYfinVqKDcMoUj9PhSkYSWPGcbCZdjdocB0P9r21SM3E+XSUc5bjnOEIpzlsWR9BTzkqVITQhW4MYAoP0pUBhNMTm2acpsSMmSTyOCwlsSwinu/VCzhGKvFkIyOjQqIT7ehFIDfSk14E0otA/HAVARqCZouMGRP5FnnLuKzQGclCpqLK+RI2VqHT4os93dEyusq+C7fVuCPVUkbQIBvQ56xGheO1eNptDiGAgjZNqR5+2AOfbYUTadDNH764XeZefTr2v8bB9HNQZIARfrB4FNwcAnYXPzYm02VSq6CIZdIZ2PqHntWrykjY64pcYY8uIghDtI6+IxO4Z4w7Y13d+MMSBYxUNRhEkpSNOTesR47+lfwtZoqL+1EsjUcnPwuAbYge14k2uIwH5xEq1M5Nb3wF5F4iekmcwoTJInud6UQfRjGVrgygM31xwKmpm10jMjLJ5HOCNI6TygnSOUEasWRQih404BiqoR8h3EB3q+h1xQ/7etRiFggaE0Xo8jCQjZFsjGRhJPsy2zmAqco1JOyryJsDfWsUOg2+qMV/cFoEQgAFbQ5ZhgNJsGAnLN6nSOCUXrYJHjEAACAASURBVPBt3zwG7IhDuiceUkoh1AWe7QF3hUO46yXXuZiz7+K+wECY9Sb42sGWDQb2bDRTmmKL2daG4pG22LyTx7DINO7s4ckwBz/UdLCeGx0NS5deek1/8nhD3k3n+cH8zQIAbP3LcB5vg98YcBoONgFN10tmwkQq54jnmFX0znCYTJQnYYeDZRg3itt5mk70IZTu2NH8Ik6UvIEFnCDNsiiid5J06/CtAzZ0xY/u+HMb/emGPxEGb46t2c3ECRPRqsRkdUHjImPCSJ5V3C4vc1kYyUWJUquMGg3eaPFGgzcafLCje5XtymKnwklIXStDCKCgzZBbouTvW7ATjqVCgBu83amAe1PO4f5VAvydCx62cFso3BMO1/nUml+ltpx9KSky0x+U6AjYdddSEm0ieFAckwamcVO7jkQ4BXHJXD9ZhtOnkbfHMGHNGYbpJDYTSTK9cSGIPrIn9u1G4TBGi+84cB4BNkEOjfESXRYZmSxSOctxznKceI5xluMkcBK9RY488KETfbiBu4mgNxH0JoiwZjdnz4iJBHI4TSanyOA0mZy0yF4h5QDYo6ULfnTDn5vpTTf86YY/7fFAVW3IyoCB4+LHUdBAzJRjJAcjudXWNffUKUJXPWeUppK8eaOlHQ70tG4r+3yst9W41Tr0KmgbCAEUtGoqjLD+pCJ+K46AbJZ50jufZepzhK1OQDqeD44amBgMb/aDG4LqDOgoLoTHH605Zx9IgEyGr54FKzdxvVd33FzCgfBKjaqAQ4cwb91N2eoMSg7aUVrWmxImY8QXgB6BhQSEZxJxlxG3MRps219b4Ssgh3iL6FVeSigEwB5HQulOJ/owgXsIpTuhdMeL5pU8MJ/SKpJ3YR1PFgbLEJcjtkTgQ1f8uJEeVtELwRO1+IEUXAFKlGupVeBMlwhdzWvZ8p+OqmiqCJsWfxzoXYfQif+ECOqPEEBBq8Nshph4ZXh36SHIL5G51ZTNJ4l5uJzMJzAjh47OuUhTguHtATA+EOxr/ygU5MKRHXAkBvZuh0MHZbLMl/uilSjLtKPd+Ym4dQTS05H37qNi3RlKt5VRdsaVElN/yngcGTskTQWOvUrwGuuI00hwHAyykwOrV8fiMaED2kYaUTRhIp0kkjhNIqdI5BRJlvWF6hlabOhAF0LpTiQ3WkWvHcGX9II1FRUYSSSXM2RWkrwMTpFprYELEIwHnfBlDJ15kuF0ph2daEeA+OEU1IAic0V1ylt1yaseEAEgYYsGLzR4WhYvbAmrtK/62hMVLuLvUtCoCAEUtArMZtifBL8dhCX7oTirgrsLUlhfeJ6zB+D54v78UilXXKCLzNybJaJvqnodWYb0JDi6Cw7HwN4YSDlheYwAE/nD1eT2kODbutsU/8hKwjMTKSvqRClDMTEFABv3Ihz6GAmYoMUpChx62yBpqw4LGwz/6OWoQhklnOeMVfIuLOc5QwV6QJmn155OhNCZ/oyiI90IozuBhKFpBl8TZVRwjmziyeYs2cSTRbxlfZ48zJbhMAds6IQvnWnHaDpbJS8cHxybcTSxoPGQMVrSlORhJA+9lI194DZyVAnIFGIiHyN5mMjDSL5lrSxgvOR6KhzQ4IXaKmy+2NHtEoGrfIwKByFzgmZH03+zCwQNpMII284oQ7srj8i4J+QzNTuFzdnnCTuTjsooszyoO7cXD75ktkxqmsS0afDjDxDuB8f3wrE98PdeKMxUjjF2MpMfpaL0JTNB3WOJtI1huLEE1dGeRH87rs72meKmkONQgkPfYnyitDiOlnEYKKH1cbnqr4WOMlI4RwrxJFuWFOJJ4gyZJFuP88afEDrTh0hu4mFCLNLnQ2CT9+gVUm6Vu7MW2bsgemkUWI9zxJYwvAnFm1vpRxg+hOJN
```

Comment:
The graph makes it look like the model fits the data rather poorly. The realizations that look like they could fit the data at the beginning of the time series seem to have way too low y_limit = b/a. This may not be a problem in the context of this notebook: as I understand it, it is designed to showcase process samples.

Reply:
Exactly. This is not an issue here.